src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -279,12 +279,12 @@ if (action === ACTIONS.STATUS || action === ACTIONS.ADD || action === ACTIONS.UP
  * @param {object} data
  * @param {string} action
  */
-async function validate_bucket_args(config_root_backend, accounts_dir_path, data, action) {
+async function validate_bucket_args(config_root_backend, accounts_dir_path, root_accounts_dir_path, data, action) {
     if (action === ACTIONS.ADD || action === ACTIONS.UPDATE) {
         if (action === ACTIONS.ADD) native_fs_utils.validate_bucket_creation({ name: data.name });
         if (action === ACTIONS.UPDATE && !_.isUndefined(data.new_name)) native_fs_utils.validate_bucket_creation({ name: data.new_name });
 
-        if (action === ACTIONS.ADD && _.isUndefined(data.bucket_owner)) throw_cli_error(ManageCLIError.MissingBucketOwnerFlag);
+        if (action === ACTIONS.ADD && _.isUndefined(data.owner_account)) throw_cli_error(ManageCLIError.MissingBucketOwnerFlag);
```

Comment:
With the new code changes it seems to fail in `get_bucket_owner_account` with an undefined name and not with the error `MissingBucketOwnerFlag`.

Reply:
I'm not sure I'm with you. Where do you see this failure? In what scenario? A test?
Are you talking about get_bucket_owner_account in fetch_bucket_data()?

Reply:
I think I meant for a manual test that you can run with creating a bucket without passing the `---owner` flag, which error do you see? (In this line it should be `MissingBucketOwnerFlag`, but I saw something else as mentioned).

Reply:
I see. Since fetch_bucket_data() is before validate_bucket_args(), I've added a check for it in fetch_bucket_data().

Reply:
I think we are trying to validate as early as we can, maybe you can do it before the `fetch_bucket_data` in a function under `validate_input_types`?
I think it might be a bit confusing to see validations in the `fetch_bucket_data`.